docs/user_guide/Certificates/Official_Certificates.inc

```diff
@@ -1,38 +1,54 @@
+2. Add the certificates for the node to key disttibution directory in the site_files.
+
+   a) Make the directory  under the key distribution directory for the clients certificates using the clients FQDN.
+   b) Copy the official certificate to that directory using the FQDN and the correct extension.
+   c) Make sure the permissions are correct.
 
-  a) Type ``mkdir -p /var/simp/environments/simp/site_files/pki_files/files/keydist/***<Client System FQDN>***``
-  b) Type
+   For example to install certificates for a system named mycomputer.my.domain:
 
-     .. code-block:: bash
+   .. code-block:: shell
 
-       mv ***<Certificate Directory>***/***<FQDN>***.[pem|pub] \
-       /var/simp/environments/simp/site_files/pki_files/files/keydist/***<FQDN>***
+     mkdir -p /var/simp/environments/simp/site_files/pki_files/files/keydist/mycomuter.my.domain
+     mv /dir/where/the/certs/were/myprivatecert.pem /var/simp/environments/simp/site_files/pki_files/files/keydist/mycomupter.my.domain.pem
+     mv /dir/where/the/certs/were/mypubliccert.pub /var/simp/environments/simp/site_files/pki_files/files/keydist/mycomupter.my.domain.pub
+     chown -R root.puppet /var/simp/environments/simp/site_files/pki_files/files/keydist
+     chmod -R u=rwX,g=rX,o-rwx /var/simp/environments/simp/site_files/pki_files/files/keydist
 
-  c) Type ``chown -R root.puppet /var/simp/environments/simp/site_files/pki_files/files/keydist``
-  d) Type ``chmod -R u=rwX,g=rX,o-rwx /var/simp/environments/simp/site_files/pki_files/files/keydist``
+3. Create and populate the certificate authority certs directory.
 
-3. Create and populate the ``/var/simp/environments/simp/site_files/pki_files/files/keydist/cacerts``
-   directory.
+   a) Make the CA directory.
+   b) Copy the root CA public certificates into cacerts in Privacy Enhanced Mail (PEM) format (one per file).
 
-  a) Type ``cd /var/simp/environments/simp/site_files/pki_files/files/keydist``
-  b) Type ``mkdir cacerts`` and copy the root CA public certificates into cacerts in Privacy
-     Enhanced Mail (PEM) format (one per file).
-  c) Type ``cd cacerts``
-  d) Type ``for file in *.pem; do ln -s $file `openssl x509 -in $file -hash -noout`.0; done``
+   .. code-block:: shell
+
+     cd /var/simp/environments/simp/site_files/pki_files/files/keydist
+     mkdir cacerts
+     cd cacerts
+     for file in *.pem; do ln -s $file `openssl x509 -in $file -hash -noout`.0; done
 
 Generating Certificates from the Fake CA
 ----------------------------------------
 
-If server certificates have not or could not be obtained at the time of client
-installation, SIMP provides a way to create them for the system, so that it
-will work until proper certificates are provided.
+The Fake (self signing) Certificate Authority (Fake CA) is provided by SIMP
+as a way to obtain server certificates if officail certificates could not be
+obtained at the time of client installation or in testing environments.
```

Comment:
'or in testing environments' -> 'or the servers are operating in a test environment'